features/step_definitions/generate.rb

```diff
@@ -1 +1,23 @@
+Then("the product {string} with destination {string} contains the framework {string} with architecture {string}") do |product, destination, framework, architecture|
```

Comment:
Here's the test @ollieatkinson

Reply:
Looking good, showing off your :ruby: skills! 

I have learnt from this, thanks! :)